Project.toml

```diff
@@ -1,6 +1,6 @@
 name = "GitHub"
 uuid = "bc5e4493-9b4d-5f90-b8aa-2b2bcaad7a26"
-version = "5.1.8"
+version = "5.1.9"
```

Comment:
Seems like a feature rather than a bugfix?

Reply:
Yeah, this can be a minor bump I suppose.